### PR TITLE
Add Sr EM competencies.

### PR DESCRIPTION
**Pull request by judahtanthony**
Add Sr EM competencies. Update EM and Director competencies.  Source is https://docs.google.com/document/d/1Yv7dRvGDhV5eosHAC4cr9S2v97CdRAmMkPBtIeSrKA8/edit#

### CODE REVIEW

File: managers.md [hunk 1]
```diff
@@ -1,115 +1,107 @@
-# Engineering Management Competencies V2
+# Engineering Management Competencies
 
-The engineering manager ladder consists of three levels, Engineering Manager, Director and VP.
+The engineering manager ladder consists of four levels:
+Engineering Manager, Senior Engineering Manager, Director, and VP.
 
 ## Engineering Manager
 
 You are a frontline manager who is responsible for a functional area and leads a small team of up to six engineers.
```
Comment (DougGrove) [inline managers.md:8]: > small team of up to six engineers.

Do we want to put an upper boundary to the size of the team for an EM?  I'll rather have something like 'small to medium sized team'

Comment (judahtanthony) [reply managers.md:8]: I agree.  We generally don't have huge teams anyway just as an organization precept.  I'll update.

File: managers.md [hunk 2]
```diff
@@ -1,115 +1,107 @@
...
+# Engineering Management Competencies
 
-The engineering manager ladder consists of three levels, Engineering Manager, Director and VP.
+The engineering manager ladder consists of four levels:
+Engineering Manager, Senior Engineering Manager, Director, and VP.
 
 ## Engineering Manager
 
-You are a frontline manager who is responsible for a functional area and leads a small team of up to six engineers.
+You are a frontline manager who is responsible for a functional area and leads a small to medium sized team.
```
Comment (hmafzal) [inline managers.md:8]: How would we size small to medium teams? I think the count is helpful. 2-6?

File: managers.md [hunk 3]
```diff
@@ -1,115 +1,107 @@
...
+- You manage & coach ICs. You look for opportunities to grow or manage new engineering managers (EMs) or Staff Engineers. You have multiple instances of successful promos.
+- You plan for team needs through suggesting headcount, proposing tools & integrations, and identifying dependencies. You understand what needs other teams within the same Organization have, and escalate or adjust if those needs will impede your team’s needs. You understand the strategy for the Organization, and ensure your team is lined up to support it.
+- You scout and attract talent. You build relationships with potential candidates. You influence org wide hiring and staffing. You moderate hiring decisions.
+- You identify macro-level patterns of team engagement, influence decisions on retention.
 
 ### Software Engineering & Design
 
-- You hold your team to the software engineering and design standards we have set for individual contributors.
-- You provide an environment (process, tools, goals, sprints, etc) that allows individual contributors to do their best work.
-- When you contribute technically, you exemplify the standards we set for individual contributors.
```
Comment (hmafzal) [inline managers.md:24]: Why remove the first and third items from this competency? Those still seem relevant

File: managers.md [hunk 4]
```diff
@@ -1,115 +1,107 @@
...
+- You escalate challenges the team is facing to the larger engineering (or engineering leadership) team, and suggest solutions.
+
+### Community & Citizenship
+
+- You own the success of your entire team. You ensure the team is celebrated for successes and has opportunities to learn from failure. 
+- You start to own at least one engineering-wide initiative (e.g. tech community pillar, interviewing, revising competencies)
+
+## Senior Engineering Manager
+
+You are an experienced manager who manages up to two teams directly or
```
Comment (hmafzal) [inline managers.md:40]: Should we add something about managing 1 large team or two teams?

File: managers.md [hunk 5]
```diff
@@ -1,115 +1,107 @@
...
-- You help create organization and structure around large projects & problems that aren’t well defined, thereby setting your team up to successfully deliver on initiatives.
-- You have established a track record of investigating and vetting the reuse of existing components and services when approaching new projects and feature requests.
-- You can orchestrate and coordinate projects & solve problems that span across multiple teams.
-- You are responsible for working with stakeholders (PM, Design, DS, etc) to drive planning for your teams OKRs and goals.
-- You partner with stakeholders (PM, Design, DS, etc) to prioritize quarterly and sprint goals as well as allocate work accordingly and effectively.
-- You provide timely code reviews and are responsive to peer feedback on code reviews and learn from it.
+- You delegate team process ownership to members of the team, load balancing as needed. You identify areas that need improvement and solicit owners from the team. You share team processes across the same Organization.
```
Comment (hmafzal) [inline managers.md:58]: Just noting the new competencies don't mention anything about established track records, and I find that a good distinction between similar competencies across different levels

File: managers.md (hunk 4 above)
Comment (judahtanthony) [reply managers.md:60]: Sure, that seems to be the implications of the EM constraints.  I'll make this change.

File: managers.md (hunk 5 above)
Comment (judahtanthony) [reply managers.md:78]: I agree that many times a "senior" in a role is largely characterized by having more experience in the base responsibilities.  My hope is that this is a generality that doesn't need to be pointed out for each common responsibility, but we can focus on the differentiating factors to keep these competencies simple and understandable.  That being said, let's resurface this conversation when we decide to update the EM and Director competencies to the new form to see if there are any areas that could use this clarifier.

**Comment by judahtanthony** [comment]
Doug gave me a "verbal" thumbs up on Slack.

**Pull request merged**